### DISCUSSION TITLE
does this support bedrock?

### DISCUSSION THREAD

**Discussion by jurgenweber** [jeremylvln/Shulker#517] [Q&A] [answered]
I went through the whole tutorial but I can not connect; we are using bedrock. Kids on tablets, etc.

note, I am very new to the minecraft world, my 5yr old just started playing it and I have gotten on board. So there is a lot of minecraft linger I am not up to speed with yet.

**Comment by jeremylvln** [comment, marked as answer]
Hi @jurgenweber!

Currently only the Java Edition is supported. There are tools that exists to allow cross-play but I've not tested myself (in standalone, not even in Shulker), so I don't have a solution for you today. I will try to invest some time learning more about Minecraft cross-play between Java and Bedrock but I can't give you a date for now.

Thanks a lot!

**Reply by jurgenweber** [reply]
Yeah, cool. Thanks for getting back to me. ALl good. Learning the eco system atm. :)
